.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
I believe you could just use an `.env.local` file. No need to have the separate `.env`.
Update: Saw your comment further down and I'll look into trying to use my own API key locally and get back to you.